MovieDBApp/MovieDBApp/Scenes/FavoritesScreen/FavoritesNavigator.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  FavoritesNavigator.swift
+//  MovieDBApp
+//
+//  Created by Phong Le on 05/08/2021.
+//
+
+import UIKit
+
+protocol FavoritesNavigatorType {
+    func toDetailScreen(movieEntity: MovieEntity)
```

Comment:
```suggestion
    func toDetailScreen(movie: Movie)
```

Em phải map data lúc lấy được rồi nhé